modules/price/src/Element/Price.php

```diff
@@ -36,7 +36,7 @@ public function getInfo() {
     $class = get_class($this);
     return [
       '#size' => 10,
-      '#maxlength' => 128,
+      '#maxlength' => NULL,
```

Comment:
Having maxlength causes utility::strlen to call which gets passed an array. @bojanz this should be fixed in 8.x-2.x HEAD right, first?